# Description: Fix some weird rendering effect at startup with compositing
#              activated
# Bug: https://bugzilla.gnome.org/show_bug.cgi?id=658036
# Bug-Ubuntu: https://bugs.launchpad.net/ubuntu/+source/metacity/+bug/820553
diff -Naur metacity-2.34.1.old//src/compositor/compositor-xrender.c metacity-2.34.1//src/compositor/compositor-xrender.c
--- metacity-2.34.1.old//src/compositor/compositor-xrender.c    2011-09-13 16:47:30.401946870 +0200
+++ metacity-2.34.1//src/compositor/compositor-xrender.c    2011-09-13 16:50:54.559380264 +0200
@@ -3090,6 +3090,8 @@
   xrc->atom_net_wm_window_type_toolbar = atoms[12];
   xrc->atom_net_wm_window_type_dropdown_menu = atoms[13];
   xrc->atom_net_wm_window_type_tooltip = atoms[14];
+  xrc->show_redraw = FALSE;
+  xrc->debug = FALSE;

 #ifdef USE_IDLE_REPAINT
   meta_verbose ("Using idle repaint\n");
